packages/frontend/web/browser/ophan/ophan.ts

```diff
@@ -38,3 +38,33 @@ export const abTestPayload = (tests: {
 
     return { abTestRegister: records };
 };
+
+export const recordPerformance = () => {
+    const performanceAPI = window.performance;
+    const supportsPerformanceProperties =
+        performanceAPI &&
+        'navigation' in performanceAPI &&
+        'timing' in performanceAPI;
+
+    if (!supportsPerformanceProperties) {
```

Comment:
Does this mean that we won't get perf data for older browsers?

Reply:
Yup, but not in any of our supported browsers - we could duplicate the previous code from Frontend but our support matrix has moved on since then?